src/Hubbup.Web/bundleconfig.json

```diff
@@ -0,0 +1,24 @@
+﻿// Configure bundling and minification for the project.
```

Comment:
No idea, it was here before and I don't know what it's used for ;). I'll try removing it.